### PR TITLE
prevent flake for user e2e

### PR DESCRIPTION
**Pull request by dsesami**
##### SUMMARY
This helps pad flake issues.

##### ISSUE TYPE

 - Bugfix Pull Request

##### COMPONENT NAME
 - UI

##### AWX VERSION

```
AWX 4.0.0
```

### CODE REVIEW

**Review by unlikelyzero — APPROVED** [review]
lotsa pauses, but we'll take a harder look at this after 3.5.0

File: awx/ui/test/e2e/objects/applications.js [hunk 1]
```diff
@@ -56,8 +56,9 @@ module.exports = {
             if (application.redirectUris) {
                 this.section.add.setValue('@redirectUris', application.redirectUris);
             }
+            this.section.add.click('@save'); // flake avoidance. triple click ensures it works.
```
Comment (elyezer) [inline awx/ui/test/e2e/objects/applications.js:59]: I wish we had waitForBeingClickable or something like that

File: awx/ui/test/e2e/tests/test-xss.js [hunk 2]
```diff
@@ -168,10 +171,8 @@ module.exports = {
         client.expect.element('#permissions_tab').visible;
         client.expect.element('#permissions_tab').enabled;
 
-        client.click('#permissions_tab');
-
-        client.expect.element('div.spinny').visible;
-        client.expect.element('div.spinny').not.visible;
+        client.pause(2000);
```
Comment (elyezer) [inline awx/ui/test/e2e/tests/test-xss.js:174]: Can't we do `client.waitForElementVisible('#permissions_tab')` here?

**Comment by dsesami** [comment]
@elyezer this is a feature that's better handled in the newest version of
nightwatch.

File: awx/ui/test/e2e/objects/applications.js (hunk 1 above)
Comment (dsesami) [reply awx/ui/test/e2e/objects/applications.js:59]: @elyezer this is a feature that's better handled in the newest version of nightwatch.

File: awx/ui/test/e2e/tests/test-xss.js [hunk 3]
```diff
@@ -237,6 +238,7 @@ module.exports = {
         client.sendKeys('div.at-Panel smart-search input', `id:>${data.notification.id - 1} id:<${data.notification.id + 1}`);
         client.waitForElementNotPresent('div.at-Panel smart-search .SmartSearch-searchButton--disabled');
         client.waitForElementNotVisible('.overlay');
+        client.pause(2000);
```
Comment (elyezer) [inline awx/ui/test/e2e/tests/test-xss.js:241]: We definitely need to check what is blocking us to click and wait until it is gone.

**Review by elyezer — COMMENTED** [review]
I know these are adding fallback actions to ensure the actions are really performed but I feel that is not the ideal way to handle it. We would want wait something to not be visible or checking if we really can click on something.

But if we can't find another option we can go ahead and merge this PR.

**Pull request merged**